rust/src/router/tests.rs

```diff
@@ -16,7 +16,10 @@ async fn init() -> Worker {
     let worker_manager = WorkerManager::new();
 
     worker_manager
-        .create_worker(WorkerSettings::default())
+        .create_worker(WorkerSettings {
+            enable_liburing: false,
+            ..WorkerSettings::default()
+        })
```

Comment:
@nazar-pc, despite this works at intended (default settings are used but `enable_liburing` which is set to `false` instead), is this correct idiomatic syntax in Rust? 

In Node the default object must be added first, then those fields whose value must be different:

```ts
options = { ...options, foo: false }
```

It surprises me that in Rust it works if the changes are added first before the object with default values.

Reply:
It is, I believe `..` should be used at the very end, I don't think it compiles otherwise at all

Reply:
Ok. For me idiomatic would be "here some values and after them those that override them" but it's ok. Rust guys have their own anti idiomatic concept of what idiomatic should mean.